package.json

```diff
@@ -49,7 +49,7 @@
     "babel-eslint": "^10.1.0",
     "colors": "^1.4.0",
     "eslint": "^7.32.0",
-    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-airbnb": "^19.0.4",
```

Comment:
https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/CHANGELOG.md#1900--2021-11-10

```
[breaking] support eslint 8; drop eslint < 7
[patch] Explain why react/jsx-key is turned off (#2474)
[fix] bump eslint-plugin-react-hooks peer dependency version (#2356)
[patch] Alphabetize the rules for react-a11y.js (#2407)
[Docs] HTTP => HTTPS (#2489)
[readme] clarify hooks requirement (#2482)
[deps] update eslint-config-airbnb-base, eslint-plugin-jsx-a11y, eslint-plugin-react, eslint-plugin-react-hooks, safe-publish-latest, eslint-plugin-import, object.entries
[meta] add --no-save to link scripts
[meta] use prepublishOnly script for npm 7+
[dev deps] update @babel/runtime, tape
```

There is actually only one breaking change here - dropping eslint less than &. It doesn't _look_ breaking for us as we use eslint `7.32.0`.

Reply:
Ahh, also, these move to `error`:

```
'react/jsx-no-script-url': ['error',

'react/jsx-no-useless-fragment': 'error',

'react/function-component-definition': ['error',

'react/jsx-no-constructed-context-values': 'error',

'react/no-unstable-nested-components': 'error',

'react/no-namespace': 'error',

'react/prefer-exact-props': 'error',

'react/no-arrow-function-lifecycle': 'error',

'react/no-invalid-html-attribute': 'error',

'react/no-unused-class-component-methods': 'error',

'react/no-unused-class-component-methods': 0,
```

https://github.com/airbnb/javascript/commit/1bc8cabd447ef9ce69d27ae70e67eb47daeffc21